src/main/java/com/replyyes/facebook/messenger/bean/FacebookMessengerSendException.java

```diff
@@ -0,0 +1,18 @@
+package com.replyyes.facebook.messenger.bean;
+
+import lombok.Getter;
+
+@Getter
+public class FacebookMessengerSendException extends Exception {
```

Comment:
Can we extend IOException instead? 

Is there a reason why we are introducing a new exception type as oppose to just reusing IOException?